tembo-operator/tests/integration_tests.rs

```diff
@@ -1292,7 +1267,18 @@ mod test {
         let params = PatchParams::apply("coredb-integration-test");
         let patch = Patch::Apply(&coredb_json);
         let _ = coredbs.patch(name, &params, &patch).await.unwrap();
-        thread::sleep(Duration::from_millis(10000));
```

Comment:
This code assumes it will finish in 10 seconds, which is not the case in a busy cluster